charts/grafana/templates/grafana-deployment.yaml

```diff
@@ -62,8 +62,6 @@ spec:
           resources:
 {{ toYaml .Values.resources | indent 12 }}
           env:
-            - name: SCHEMA_NAME
-              value: "grafana"
```

Comment:
@danielhoherd @rishkarajgi as per the code this schema is auto applied by grafana source code